mail_message_icon/wizard/mail_resend_message.py

```diff
@@ -0,0 +1,49 @@
+from odoo import Command, api, fields, models
+
+
+class MailResendMessage(models.TransientModel):
+    _inherit = "mail.resend.message"
+
+    @api.model
+    def default_get(self, fields):
+        """Overridden: Add status field for a resend message wizard."""
+        message_id = self._context.get("mail_message_to_resend")
+        mode = self._context.get("mail_status_mode", False)
+        if not message_id or not mode:
+            return super(MailResendMessage, self).default_get(fields)
+        else:
```

Comment:
You can remove `else` and remove indent of all code below. It either gonna miss `message_id` or `mode` and return parent implementation of `default_get` method or proceed with everything else anyway